src/frontend/web_application/src/layouts/Page/components/Navigation/components/StickyNavBar/index.jsx

```diff
@@ -25,8 +25,11 @@ class StickyNavbarClass extends Component {
 
   handleScroll(ev) {
```

Comment:
That function is called way too many times, you'll need to either throttle or debounce it ..
See https://css-tricks.com/debouncing-throttling-explained-examples/ for examples